lochness/mindlamp/__init__.py

```diff
@@ -215,6 +330,10 @@ def get_sensor_events_lamp(lamp: LAMP, subject_id: str) -> List[dict]:
     Returns:
         activity_dicts: activity records, list of dict.
     '''
-    sensor_event_dicts = lamp.SensorEvent.all_by_participant(subject_id)['data']
-
+    if not from_ts == None:
```

Comment:
This should be just `if from_ts`:

Reply:
I'll change it to `if val is not None:` as noted in the link below (the answer at the top)
https://stackoverflow.com/questions/3965104/not-none-test-in-python

Reply:
aca77b9